cherryontech/unmatched
# #UM 14 Bio Display Page

## Description
This PR adds a "bio page" where the user can view detailed information about a specific individual.



## Related Tickets & Documents
[Fixes UM #14](https://lemon-zest.atlassian.net/jira/software/projects/UM/boards/2?selectedIssue=UM-14)

## Acceptance Criteria
- [x] Clicking on a profile card from the Professional page navigates to the individual's "Bio Display Page".
- [x] The "Bio Display Page" displays detailed information about the individual, including their full bio, photo, title, and associated resources.
- [x] Navigation options are available to return to the previous page or browse other resources.

## What type of PR is this? (check all applicable)


|     | Type                       |
| --- | -------------------------- |
|✓| :sparkles: New feature     |
|     | 🎨 Style                   |
|     | :hammer: Refactoring       |
|     | 🔥 Performance Improvements|
|     | :bug: Bug fix              |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Mobile & Desktop Screenshots/Recordings

https://www.loom.com/share/1eaad8020a844c75a3c51cedfc5e8958?sid=4d233f42-aef5-4276-98ef-64ef4499e2e4

## What I learned

I learned more about how to style with grid. Flexbox is usually my preference, so I dont use grid often, but I tried to use grid when styling the layout of professionals cards. Resources that came in handy are: [A Complete Guide to CSS Grid](https://css-tricks.com/snippets/css/complete-guide-grid/) and [A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)

## Testing steps
1. Install dependencies: `npm i`
2. Run `git checkout feature/UM-14-bio-page`
3. Run localhost: `npm run dev`
4. View the app in browser and navigate to `/professionals` page. Press a professionals card and you'll see the "bio page" (e.g., `/professionals/1/Reshma%20Saujani`) with information about them, resources, and other recommended professionals.


## Added to documentation?

|     | Type                       |
| --- | -------------------------- |
|     | 📜 README.md    |
|     | 🙅 no documentation needed    |


## What gif best describes this PR or how it makes you feel?
![Boy from home alone movie saying yes](https://media.giphy.com/media/xUySTUZ8A2RJBQitEc/giphy.gif)
